src/NuGetPackageVerifier/Rules/AssemblyStrongNameRule.cs

```diff
@@ -13,6 +15,8 @@ namespace NuGetPackageVerifier.Rules
 {
     public class AssemblyStrongNameRule : IPackageVerifierRule
     {
+        private static byte[] _publicKeyTokenBytes = new byte[] { 173, 185, 121, 56, 41, 221, 174, 96 };
```

Comment:
Actually can this be in hex? I think that would look more familiar to someone seeing it.